docs/specification.md

```diff
@@ -917,15 +1761,72 @@ When an agent supports multiple protocols, all supported protocols **MUST**:
-| Cancel task         | `tasks/cancel`                       | `CancelTask`           | `POST /v1/tasks/{id}:cancel`      |
-| Resubscribe to task | `tasks/resubscribe`                  | `TaskResubscription`   | `POST /v1/tasks/{id}:resubscribe` |
-| Get agent card      | `agent/getAuthenticatedExtendedCard` | `GetExtendedAgentCard` | `GET /v1/extendedAgentCard`       |
+| Functionality                        | JSON-RPC Method                          | gRPC Method                          | REST Endpoint                                        |
+| :----------------------------------- | :--------------------------------------- | :----------------------------------- | :--------------------------------------------------- |
+| Send message                         | `message/send`                           | `SendMessage`                        | `POST /v1/message:send`                              |
+| Stream message                       | `message/stream`                         | `SendStreamingMessage`               | `POST /v1/message:stream`                            |
+| Get task                             | `tasks/get`                              | `GetTask`                            | `GET /v1/tasks/{id}`                                 |
+| List tasks                           | `tasks/list`                             | `ListTasks`                          | `GET /v1/tasks`                                      |
+| Cancel task                          | `tasks/cancel`                           | `CancelTask`                         | `POST /v1/tasks/{id}:cancel`                         |
+| Resubscribe to task                  | `tasks/resubscribe`                      | `TaskResubscription`                 | `POST /v1/tasks/{id}:resubscribe`                    |
+| Set push notification config         | `tasks/pushNotificationConfig/set`       | `SetTaskPushNotificationConfig`      | `POST /v1/tasks/{id}/pushNotificationConfigs`        |
+| Get push notification config         | `tasks/pushNotificationConfig/get`       | `GetTaskPushNotificationConfig`      | `GET /v1/tasks/{id}/pushNotificationConfigs/{configId}` |
+| List push notification configs       | `tasks/pushNotificationConfig/list`      | `ListTaskPushNotificationConfig`     | `GET /v1/tasks/{id}/pushNotificationConfigs`         |
+| Delete push notification config      | `tasks/pushNotificationConfig/delete`    | `DeleteTaskPushNotificationConfig`   | `DELETE /v1/tasks/{id}/pushNotificationConfigs/{configId}` |
+| Get extended Agent Card | `agent/getExtendedAgentCard`     | `GetExtendedAgentCard`               | `GET /v1/extendedAgentCard`                          |
+
+### 5.4. JSON Field Naming Convention
+
+All JSON serializations of the A2A protocol data model **MUST** use **camelCase** naming for field names, not the snake_case convention used in Protocol Buffer definitions.
```

Comment:
I would suggest that we specifically call out using: https://protobuf.dev/programming-guides/json/ 

This already declares: `Message field names are mapped to lowerCamelCase and become JSON object keys.` etc. 

It also has a large section about RFC3339 timestamps etc.

If we declare we're using ProtoJSON then it's safe in the SDKs to use the protobuf JSON encoders / decoders which exist for a bunch of different language bindings. The current proposal we'll need to implement custom JSON conversion logic.

Reply:
bump